File: Typescript Codewars/8kyu/String Templates - Bug Fixing #5.ts
/*
Description

Oh no! Timmy hasn't followed instructions very carefully and forgot how to use the new String Template feature, Help Timmy 
with his string template so it works as he expects!

export function buildString(... template: string[]): string {
  return `I like {template.join(',')}!`
}
*/

export function buildString(...template: string[]): string {
  return `I like ${template.join(", ")}!`;
}

File: Java Codewars/8kyu/String Templates - Bug Fixing #5.java
/*
Description

Oh no! Timmy hasn't followed instructions very carefully and forgot how to use the new String Template feature, Help Timmy with 
his string template so it works as he expects!

public class Kata 
{
    public static String buildString(String... args)
    {
      return
    }
}
*/

public class Kata {

  public static String buildString(String... args) {
    return "I like " + String.join(", ", args) + "!";
  }
}
